### ISSUE TITLE
Support Rails 5.1.1

### ISSUE THREAD

**Issue by clairity** [gregnavis/active_record_doctor#12]
i'm on `rails 5.1.1` (ruby 2.4.1) and it seems bundler resolves active_record_doctor to `1.0.1`, which depends on `rails 4.2.x`:

```active_record_doctor was resolved to 1.0.1, which depends on rails (~> 4.2)```

if i track master, then it resolves to `1.3.0`, but neither seem to support `rails 5.1.x`:

```active_record_doctor was resolved to 1.3.0, which depends on rails (< 5.1, >= 4.2)```

an update would be appreciated! 😊

**Comment by gregnavis** [comment]
Thank you for reporting, @clairity. I'll look into this soon.

**Comment by gregnavis** [comment]
@clairity, I just resolved the issue and added official support for Rails 4.2, 5.0 and 5.1. Could you confirm it works in your project? If so, I'll release a new version of the gem. You can pull in from GitHub by adding the following to `Gemfile`:

```ruby
group :development do
  gem 'active_record_doctor', git: 'https://github.com/gregnavis/active_record_doctor.git'
end
```

**Comment by clairity** [comment]
@gregnavis works great, thanks! btw, i used this shortcut form in the gemfile since i have set `bundle config github.https true`:
```ruby
gem 'active_record_doctor', github: 'gregnavis/active_record_doctor'
```

**Comment by gregnavis** [comment]
Fixed in d20798ec1c526b445f519df1b54b37d368997b0a.
